sharktuner/sharktuner/libtuner.py

```diff
@@ -1315,6 +1334,17 @@ def benchmark(
         candidate_results,
         prune_slow_candidates=tuning_client.should_prune_slower_candidates(),
     )
+    if all_candidates_with_speedup:
+        for i, handler_res in enumerate(all_candidates_with_speedup, start=1):
```

Comment:
Do we need the if condition? I think if the list is empty, the for loop won't execute?